client/components/social-buttons/style.scss

```diff
@@ -10,3 +10,18 @@
 		max-width: 200px;
 	}
 }
+
+.apple__signin-button div {
```

Comment:
Could you adjust this class name to match Calypso naming conventions? (https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md)

Reply:
Linter said `apple__` or `social-buttons__` was fine. Switched to `social-buttons__apple-button` as I think that is most in line with the convention.